src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,23 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	// Using stripos() to allow for multiple occurrences ("section_start_1", "section_start_2" etc).
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
+		echo '<div class="tribe-events-widget-admin-form__input-section">';
+		if ( ! empty( $field['title'] ) ) {
+			// Note: the actual widget title/handle is an <h3>
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 68dc2d19d787cadceeec1fa54bd1ee1f</sub>